src/nlCommandHandler.ts

```diff
@@ -1,7 +1,19 @@
 import { Message } from 'discord.js';
 import { findMember } from './bamands';
 import { transferredPoints } from './database/allModels';
-import { hogwartsHouse } from './types/enums';
+import { hogwartsHouses } from './shared_assets';
+
+async function getHouseOfUser(memberID: string, msg: Message) {
+  const member = await msg.guild?.members.fetch(memberID);
```

Comment:
not sure if `guild?.` will cause problems because I'm not sure if Node 14 already supports this shorthand. If it doesn't, we will need to check for msg.guild first ( which we already do before calling nlCommandhandler IIRC, but we can't just assume that. )